docs/Content-Reuse.md

```diff
@@ -20,8 +20,33 @@ Site variables are defined within `jekyll/_config.yml` as key value pairs:
 my-variable-name: my-variable-value
```

Comment:
@catherinepope I think we also actually need to separate out how to add the variables to the `_config` file. SO for asciidoc they need defining as attributes under `asciidoctor:`

Reply:
Wouldn't they only go under `asciidoctor` for PDFs? I can't see anywhere to add a non-AsciiDoc attribute - where would they go?